Lib/test/test_pyexpat.py

```diff
@@ -755,5 +755,73 @@ def resolve_entity(context, base, system_id, public_id):
         self.assertEqual(handler_call_args, [("bar", "baz")])
 
 
+class ReparseDeferralTest(unittest.TestCase):
+    def test_getter_initial_value(self):
+        if expat.version_info >= (2, 6, 0):
+            expected = True
+        else:
+            expected = False
+
+        parser = expat.ParserCreate()
+        actual = parser.GetReparseDeferralEnabled()
+
+        self.assertEqual(actual, expected)
+
+    def test_getter_setter_round_trip(self):
+        parser = expat.ParserCreate()
+        enabled = (expat.version_info >= (2, 6, 0))
+
```

Comment:
Please remove redundant empty lines, they make reading more difficult. Empty lines are needed between functions, classes, etc, and in a function they are used to separate large semantically distinct groups of code if the function is large.

This method is not large enough to use empty lines.

Reply:
@serhiy-storchaka subjective for sure, but dropped, done.